pkg/chain/local/local.go

```diff
@@ -145,6 +179,9 @@ func (lc *localChain) SubmitKeepPublicKey(
 	keepAddress common.Address,
 	publicKey [64]byte,
 ) error {
+	lc.handlerMutex.Lock()
+	defer lc.handlerMutex.Unlock()
```

Comment:
Ok sure. But, I can't rename it to `mutex` because there is also a `mutex` instance in the `TBTCLocalChain` which extends the `localChain`. I will do the following:
- rename `handlerMutex` in `localChain` to `localChainMutex`
- rename `mutex` in `TBTCLocalChain` to `tbtcLocalChainMutex`